server/utils.go

```diff
@@ -166,3 +166,41 @@ func parseJIRAIssuesFromText(text string, keys []string) []string {
 
 	return issues
 }
+
+func isImageMIME(mime string) bool {
+	return strings.HasPrefix(mime, "image")
+}
+
+func isEmbbedableMIME(mime string) bool {
+	switch mime {
+	// .swf
+	case "application/x-shockwave-flash":
+		return true
+	// .mov
+	case "video/quicktime":
+		return true
+	// .rm
+	case "application/vnd.rn-realmedia":
+		return true
+	// .ram
+	case "audio/x-pn-realaudio":
+		return true
+	// .mp3
+	case "audio/mpeg3":
+		return true
+	case "audio/x-mpeg-3":
+		return true
+	case "video/mpeg":
+		return true
+	case "video/x-mpeg":
+		return true
+	// .wmv
+	case "video/x-ms-wmv":
+		return true
+	// .wma
+	case "video/x-ms-wma":
+		return true
+	default:
+		return false
```

Comment:
Thanks for putting this together. 1/5 These should probably go into an array. Where did this list come from?

Reply:
Sure, I can put them in an array.
The list has two "sources". First one would be Jira documentation on embeddable objects (when creating an issue, the `?` button below the description box). On that list you would have the extensions of the files permitted. Since I felt it was better to check mime than file extensions, I searched the web for the mime extensions, taking most of them from [here](https://www.sitepoint.com/mime-types-complete-list/)

Reply:
Okay since the second source is not from Jira, I would think we want to test these out. Though I think the mime types are more-or-less portable.